boards/nucleo-l412kb/Makefile.features

```diff
@@ -0,0 +1,21 @@
+CPU = stm32l4
+CPU_MODEL = stm32l412kb
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_pwm
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_rtt
+FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_timer
+FEATURES_PROVIDED += periph_uart
+
+# Put other features for this board (in alphabetical order)
+# Note that a recent version of OpenOCD is required for flashing an image on
```

Comment:
I think this comment is not in sync with that board support since you need a specific patch of OpenOCD.